packages/client/mutations/AcceptTeamInvitationMutation.ts

```diff
@@ -180,19 +178,29 @@ const AcceptTeamInvitationMutation: StandardMutation<
       const {authToken, team} = acceptTeamInvitation
       const serverError = getGraphQLError(data, errors)
       if (serverError) {
-        if (serverError.message === InvitationTokenError.ALREADY_ACCEPTED) {
+        const message = serverError.message
+        if (message === InvitationTokenError.ALREADY_ACCEPTED) {
           handleAuthenticationRedirect(acceptTeamInvitation, {
             atmosphere,
             history,
             meetingId: locallyRequestedMeetingId
           })
+        } else if (!ignoreApproval) {
+          atmosphere.eventEmitter.emit('addSnackbar', {
```

Comment:
-1 can we move this logic to `handleAcceptTeamInvitationErrors` since things like loginwithGoogle also call acceptTeamInvitation (see https://github.com/ParabolInc/parabol/blob/218c326c9f6c50f9d1496f2b25041a628bfa0a40/packages/client/mutations/LoginWithGoogleMutation.ts#L28)

as it's written if someone who is logged out got an invite link, pasted it, logged in with google, i don't think they would see the error message, would they?

Reply:
I just double checked and they will still see the locked error message as we return the error from the server and show the message here: https://github.com/ParabolInc/parabol/blob/ee58e1899bb0f04c06921b022e5b4025a94ce4ab/packages/client/mutations/AcceptTeamInvitationMutation.ts#L151

I left a comment here https://github.com/ParabolInc/parabol/pull/6900#discussion_r930911479 mentioning why I'd prefer to separate the logic. If you'd still prefer it to be added in, just let me know!

Reply:
ahhh gotcha. The only difference I see is that if you're not logged in you don't get the "Contact Sales" CTA, but not a show stopper!